tests/utils/Statistic/min-intervals.test.js

```diff
@@ -0,0 +1,23 @@
+import { describe, expect, test } from 'vitest'
+import getMinInterval from '$lib/utils/Statistic/min-interval';
+import data from './min-intervals-data.json' assert { type: 'json' };
+
+describe("Find smallest interval", () => {
+    test("should return the smallest interval", () =>{
+        expect(getMinInterval(data)?.intervalString).toBe("5m")
+    })
+})
+
+describe("Find smallest interval between forecasts", () => {
+    test("should return smallest interval", () => {
+
+        expect(getMinInterval(data, 'forecast')?.intervalString).toBe("5m")
+    })
+})
+
+describe("Find smallest interval between histories since statType is not provided", () => {
```

Comment:
this test looks like the same one as the first test.
maybe should be:
```
expect(getMinInterval(data, 'history')?.intervalString).toBe('5m');
```

Reply:
I've fixed it